packages/wallet/frontend/src/pages/account/[accountId].tsx

```diff
@@ -60,7 +60,7 @@ const AccountPage: NextPageWithLayout<AccountPageProps> = ({
             }
             className="group flex aspect-square h-24 w-24 flex-col items-center justify-center -space-y-1 rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6"
           >
-            <New className="h-7 w-7" />
+            <New className="h-9 w-7" />
```

Comment:
https://github.com/interledger/testnet/pull/409#discussion_r1231984219 👈

Reply:
I somehow missed this, sorry :D